pkg/sidecarconfig/sidecarconfig.go

```diff
@@ -199,3 +201,9 @@ func TemplateSanityCheck(sidecarConfigTemplate *template.Template) error {
 	_, err := RenderTemplate(dummyPod, sidecarConfigTemplate)
 	return err
 }
+
+func SidecarTemplateExtraFuncs() template.FuncMap {
+	return template.FuncMap{
+		"fromYaml": templates.FromYAML,
```

Comment:
Related question.. since YAML parses JSON, I presume this works with JSON, too?

Reply:
Yes, this will permit parsing JSON as well as YAML.